.gitignore

```diff
@@ -12,3 +12,11 @@
 hs_err_pid*
 
 /target/
+
+# eclipse
+.classpath
+.settings
+core/.classpath
```

Comment:
you probably don't need these last four lines. I think the two above should do the trick